src/server/socketlistener.cpp

```diff
@@ -41,6 +41,7 @@ SocketListener::SocketListener(QIODevice *socket) : socket(socket) {
 void SocketListener::sendMessage(const QString &message, bool doWait) const {
     if (_threadId != std::this_thread::get_id()) {
         LOGW_ERROR(KDC::Log::instance()->getLogger(), L"SocketListener::sendMessage should only be called from the main thread");
```

Comment:
Unless you absolutely want to log an error message, you can use a one line assertion with a message as depicted in this SO post: https://stackoverflow.com/questions/3692954/add-custom-messages-in-assert.

Reply:
No issue for me to remove this log line: [627464b](https://github.com/Infomaniak/desktop-kDrive/pull/238/commits/627464bb363dc2acb289bd7944b302b5e2398445)